### ISSUE TITLE
[argo-ui] EventSource MIME type error when trying to view logs

### ISSUE THREAD

**Issue by discordianfish** [argoproj/argo#1263]
**BUG REPORT**:

**What happened**:
Getting this error when trying to view the logs from argo-ui:

```
EventSource's response has a MIME type ("application/json") that is not "text/event-stream". Aborting the connection.
```

Same result whether I'm using an ingress and reverse proxy or kubectl port-forward.

**Environment**:
- Argo version: v2.2.1

**Comment by jessesuen** [comment]
@discordianfish we think this is solved here:
https://github.com/argoproj/argo/issues/1114#issuecomment-444741682
Can you try with argoproj/argoui:v2.2.1-37808b1 ?

**Comment by discordianfish** [comment]
@jessesuen Looks like it's fixed in v2.2.1-37808b1, thanks!
When will you do the next release? There are a few fixes in master that haven't been released. Cutting a v2.2.2 shouldn't take too much time, right? Can I help with that?
